library/src/main/scala/scala/scalajs/niocharset/UTF_8.scala

```diff
@@ -52,6 +53,7 @@ private[niocharset] object UTF_8 // scalastyle:ignore
       // > 11110111
       -1, -1, -1, -1, -1, -1, -1, -1
   )
+  // format: on
```

Comment:
Yes, this is perfectly warranted (hence no comment referencing a scalafmt issue).